### PR TITLE
Workbench: Remove not needed explicit OK and CANCEL button

### PR DESCRIPTION
**Pull request by frlan**

### CODE REVIEW

File: workbench/src/dialogs.c [hunk 1]
```diff
@@ -289,8 +289,8 @@ gboolean dialogs_directory_settings(WB_PROJECT_DIR *directory)
 	dialog = gtk_dialog_new_with_buttons(_("Directory settings"),
 										 GTK_WINDOW(wb_globals.geany_plugin->geany_data->main_widgets->window),
 										 flags,
-										 _("_Cancel"), GTK_RESPONSE_CANCEL,
-										 _("_OK"), GTK_RESPONSE_ACCEPT,
+										 GTK_STOCK_CANCEL, GTK_RESPONSE_CANCEL,
+										 GTK_STOCK_OK, GTK_RESPONSE_ACCEPT,
```
Comment (LarsGit223) [inline workbench/src/dialogs.c]: As per the documentation under https://developer.gnome.org/gtk3/stable/gtk3-Stock-Items.html#GTK-STOCK-CANCEL:CAPS stock items are deprecated. I think it should better stay as-is.

Comment (frlan) [reply workbench/src/dialogs.c]: Puh. Whatever is behind this I don't like it. :/

Comment (LarsGit223) [reply workbench/src/dialogs.c]: Found this blog which says it was deprecated to save memory, see https://blogs.gnome.org/tko/2005/10/26/gtk-builtin-stock-icons-removed/. On this quick search I did not find anything from the gtk development team. Anyway, we can't change it.

Comment (codebrainz) [reply workbench/src/dialogs.c]: IIRC this was part of the [Infamous Deprecations of 2013](https://mail.gnome.org/archives/gtk-devel-list/2013-July/msg00000.html). I believe the issue of each project having to provide their own translations for each of these stock items was brought up and dismissed by GNOME designers.

**Pull request closed without merging**